src/test/kotlin/com/linchpino/core/service/InterviewServiceTest.kt

```diff
@@ -47,36 +47,34 @@ class InterviewServiceTest {
     private lateinit var service: InterviewService
 
     @Mock
-    private lateinit var interviewRepo: InterviewRepository
+    private lateinit var interviewRepository: InterviewRepository
 
     @Mock
-    private lateinit var accountRepo: AccountRepository
+    private lateinit var accountRepository: AccountRepository
 
     @Mock
-    private lateinit var jobPositionRepo: JobPositionRepository
+    private lateinit var jobPositionRepository: JobPositionRepository
 
     @Mock
-    private lateinit var interviewTypeRepo: InterviewTypeRepository
+    private lateinit var interviewTypeRepository: InterviewTypeRepository
 
     @Mock
-    private lateinit var timeSlotRepo: MentorTimeSlotRepository
+    private lateinit var mentorTimeSlotRepository: MentorTimeSlotRepository
 
     @Mock
     private lateinit var accountService: AccountService
 
-//    @Mock
-//    private lateinit var roleRepository: RoleRepository
-
     @Test
     fun `test create new interview when account exists`() {
-        // Given
         val jobSeekerAccount = Account().apply {
-            id = 2
+            id = 1
```

Comment:
Having two accounts with the same ID is semantically incorrect, although you are not persisting them in the database.